app/serializers/email_serializer.rb

```diff
@@ -1,35 +1,11 @@
 # frozen_string_literal: true
 
 class EmailSerializer < ActiveModel::Serializer
-  attribute :email
-  attribute :effective_at
+  attribute :email_address, key: :email
+  attribute :effective_date, key: :effective_at
```

Comment:
@hpjaj reverted serializer keys

Reply:
Prior to this there was one level of re-mapping, from EVSS::PCIU's attrs to ours.  This change will add a second layer of re-mapping.  

It would now go from EVSS::PCIU attrs in the response, which are then remapped in our custom response, then remapped again in the serializer.  

Perhaps just revert all of these re-naming changes?